From 285c7ac5d5027d45bbcdc49334243f2a3ba6dd05 Mon Sep 17 00:00:00 2001
From: Aleksa Sarai <asarai@suse.de>
Date: Mon, 5 Nov 2018 21:12:41 +1100
Subject: [PATCH] makefile: remove emoji

These break the build on i386 in some very specific circumstances (where
/bin/sh breaks when handling unicode), and are completely useless when
it comes to actually building the project.

Signed-off-by: Aleksa Sarai <asarai@suse.de>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: containerd-1.3.7_8fba4e9a7d01/Makefile
===================================================================
--- containerd-1.3.7_8fba4e9a7d01.orig/Makefile
+++ containerd-1.3.7_8fba4e9a7d01/Makefile
@@ -60,8 +60,8 @@ else
 	DEBUG_TAGS := static_build
 endif
 
-WHALE = "ðŸ‡©"
-ONI = "ðŸ‘¹"
+WHALE = "[w]"
+ONI = "[o]"
 
 RELEASE=containerd-$(VERSION:v%=%).${GOOS}-${GOARCH}
 
